app/Models/Internet/MacAddress.php

```diff
@@ -33,15 +55,49 @@ class MacAddress extends Model
         'user_id', 'mac_address', 'comment', 'state', 'ip',
     ];
 
+    // Default attributes
     protected $attributes = [
-        'comment' => '',
         'state' => self::REQUESTED,
     ];
```

Comment:
What does this do exactly? Also it would be nice to use enums instead of const strings.

Reply:
Default values are defined here instead of DB level. We don't have a convention here. I created a new issue for enums as we need to change it everywhere.

Reply:
But you can still do it here. Less to refactor later :D